pacta.loanbook.Rproj

```diff
@@ -1,4 +1,5 @@
 Version: 1.0
+ProjectId: aef3dc3e-b723-403b-b3d1-fbf024e0aca5
```

Comment:
This is a new thing that RStudio is doing. If we set it once, it should be used by any clone of this repo, so we'll never have to set it again.

Reply:
If one uses a version of RStudio older than RStudio 2024.12.0, apparently it may keep trying to remove this line

Reply:
uh-oh, time for me to update RStudio I guess